src/redis-cli.c

```diff
@@ -2168,13 +2175,17 @@ static void repl(void) {
 
 static int noninteractive(int argc, char **argv) {
     int retval = 0;
+
+    char **converted_argv = convertToSds(argc, argv);
```

Comment:
i would have actually preferred to do this:

```c
argv = convertToSds(argc, argv);
```
since it would men not modifying all the lines with the rename of converted_argv.

but if you prefer to create a new var let's change it's type.
```suggestion
    sds *converted_argv = convertToSds(argc, argv);
```
and if we do that, we should also change the type of the input argument for the `freeConvertedSds` and the return type of `convertToSds` to be `sds *`.
and then change the variable name to in them to `args`, rather than `sds` which conflicts with the type (confusing)

Reply:
I prefer the first one.
Fixed and rebased.